pytest.ini

```diff
@@ -8,6 +8,6 @@
 
 [pytest]
 pep8ignore = docs/conf.py ALL
-addopts = --pep8 --doctest-glob="*.rst" --doctest-modules --cov=invenio_search --cov-report=term-missing
+addopts = --doctest-glob="*.rst" --doctest-modules --cov=invenio_search --cov-report=term-missing
```

Comment:
Minor: I think you need to add ``--pycodestyle`` to have pep8 check running.

Reply:
:+1: moved to use `pytest-invenio` and all the new options